pkg/apis/networkaddonsoperator/v1alpha1/networkaddonsconfig_types.go

```diff
@@ -9,13 +9,21 @@ import (
 // NetworkAddonsConfigSpec defines the desired state of NetworkAddonsConfig
 // +k8s:openapi-gen=true
 type NetworkAddonsConfigSpec struct {
-	Multus          *Multus           `json:"multus,omitempty"`
-	LinuxBridge     *LinuxBridge      `json:"linuxBridge,omitempty"`
-	Ovs             *Ovs              `json:"ovs,omitempty"`
-	KubeMacPool     *KubeMacPool      `json:"kubeMacPool,omitempty"`
-	ImagePullPolicy corev1.PullPolicy `json:"imagePullPolicy,omitempty"`
-	NMState         *NMState          `json:"nmstate,omitempty"`
-	MacvtapCni      *MacvtapCni       `json:"macvtap,omitempty"`
+	Multus           *Multus           `json:"multus,omitempty"`
+	LinuxBridge      *LinuxBridge      `json:"linuxBridge,omitempty"`
+	Ovs              *Ovs              `json:"ovs,omitempty"`
+	KubeMacPool      *KubeMacPool      `json:"kubeMacPool,omitempty"`
+	ImagePullPolicy  corev1.PullPolicy `json:"imagePullPolicy,omitempty"`
+	NMState          *NMState          `json:"nmstate,omitempty"`
+	MacvtapCni       *MacvtapCni       `json:"macvtap,omitempty"`
+	CertificateKnobs *CertificateKnobs `json:"certificateKnobs,omitempty"`
```

Comment:
That is pretty informal name. What about  `SelfSignConfiguration`, to follow KubeVirt's API?

Reply:
Done.